.github/workflows/release.yml

```diff
@@ -51,10 +77,25 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      
+      - name: Login to Quay
+        uses: docker/login-action@v1
+        with:
+          registry: quay.io
+          username: ${{ secrets.QUAY_USERNAME }}
+          password: ${{ secrets.QUAY_TOKEN }}
 
       - name: Build & Push Image
-        env:
-          IMAGE_ORG: ${{ secrets.IMAGE_ORG}}
-        run: |
-          make docker.buildx.csi-driver
-          make buildx.push.csi-driver
+        uses: docker/build-push-action@v2
```

Comment:
how does it know which make command to run?

Reply:
This does not require any make file. It simulates the command `docker buildx -t image-name:tag -f path/to/dockerfile` and all the command args are fields in the action.

Reply:
You can see it has the tag, build-arg, dockerfile, etc fields.